pkgs/development/mobile/androidenv/mkrepo.sh

```diff
@@ -6,14 +6,16 @@ set -e
 pushd "$(dirname "$0")" &>/dev/null || exit 1
 
 echo "Writing repo.json" >&2
-ruby mkrepo.rb \
+REPO_JSON="$(cat ./repo.json)"
+echo $REPO_JSON | ruby mkrepo.rb \
     --packages ./xml/repository2-1.xml \
     --images ./xml/android-sys-img2-1.xml \
     --images ./xml/android-tv-sys-img2-1.xml \
     --images ./xml/android-wear-cn-sys-img2-1.xml \
     --images ./xml/android-wear-sys-img2-1.xml \
     --images ./xml/google_apis-sys-img2-1.xml \
     --images ./xml/google_apis_playstore-sys-img2-1.xml \
-    --addons ./xml/addon2-1.xml > repo.json
+    --addons ./xml/addon2-1.xml \
+         > repo.json
```

Comment:
```suggestion
    --addons ./xml/addon2-1.xml \
      | sponge repo.json
```

Reply:
I got 
```
./mkrepo.sh: line 18: sponge: command not found
```
and apparently `sponge` is not available in `nixpkgs`! Let me know how can I resolve it.

Reply:
sponge is part of moreutils

```
 ▶ nix-locate bin/sponge
moreutils.out                                    18,400 x /nix/store/5zh3jp0q5w108111ppcwjxrpbxsy3wsl-moreutils-0.67/bin/sponge
```

Reply:
Thanks, fixed in https://github.com/NixOS/nixpkgs/pull/227666/commits/de2679819e0ab01b9488b151efe4a2d433f1fa25 It also solved the problem with empty stdin above!